back-end/student-tracker-template.yaml

```diff
@@ -39,6 +39,33 @@ Resources:
       Role: !GetAtt IAMRole.Arn
     DependsOn:
       - DynamoDBTable
+  CoursesFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Code:
+        ZipFile: |
+          const AWS = require('aws-sdk');
+          const docClient = new AWS.DynamoDB.DocumentClient();
+          const params = {
+           TableName: !Sub "next-byte-Courses-${Env}",
+           KeyConditionExpression: 'my-partition-key = :pk',
+           ExpressionAttributeValues: {
+            ':pk': 'my-partition-key-value'
+           }
+          };
+          docClient.query(params, function(err, data) {
+           if (err) {
+             console.log(err);
```

Comment:
it will be more realistic to catch errors and send out a user-friendly error message ( if necessary) ,rather than log the raw error. console.log is better during debugging of code